src/CONST.ts

```diff
@@ -5287,24 +5287,53 @@ const CONST = {
     },
 
     EXCLUDE_FROM_LAST_VISITED_PATH: [SCREENS.NOT_FOUND, SCREENS.SAML_SIGN_IN, SCREENS.VALIDATE_LOGIN] as string[],
-
     EMPTY_STATE_MEDIA: {
         ANIMATION: 'animation',
         ILLUSTRATION: 'illustration',
         VIDEO: 'video',
     },
-
-    UPGRADE_FEATURE_INTRO_MAPPING: [
-        {
+    UPGRADE_FEATURE_INTRO_MAPPING: {
+        reportFields: {
             id: 'reportFields',
             alias: 'report-fields',
             name: 'Report Fields',
             title: 'workspace.upgrade.reportFields.title',
             description: 'workspace.upgrade.reportFields.description',
             icon: 'Pencil',
         },
-    ],
-
+        netSuite: {
+            id: 'netSuite',
```

Comment:
Let's avoid using `netSuite` in this camel case style always. Use `CONST.POLICY.CONNECTIONS.NAME.NETSUITE` if you need this.

Reply:
Handled.